### ISSUE TITLE
Programmatic name is not given for "Delete" button: A11y_Azure Tools Storage Explorer_Manage Stored Access Policies_Devtools

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/AzureStorageExplorer#8410]
### Preflight Checklist

- [x] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [x] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [x] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.36.2

### Architecture

x64

### Platform

Windows

### OS Version

Windows 11

## GitHub Tags:
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #BM_AzureToolsStorageExplorer_Win32_Dec2024; #Azure Tools Storage Explorer; #Devtools; #WCAG4.1.2; #FTP; #Closed; #Regressed:02-18-25;

## Environment Details:
Application: Microsoft azure storage explorer
Version: 1.36.2
OS: Windows 11 Enterprise 24H2
Build: 26100.2605

## Repro Steps

1. Launch Azure Storage Explorer application.
2. TAB till "Account Management" in left Navigation and press ENTER key.
3. TAB till any "Subsection" for example 'azure tools Manual testing 1' and select it.
4. TAB till "Open explorer" button and press ENTER key
5. "Explorer" section will open. TAB till selected subscription and expand it
6. TAB till storage account and expand it.
7. TAB till any storage account and expand it.
8. TAB till "Blob Containers" and press ENTER key to expand it.
9. TAB till any "Blob container" and press ENTER key to select it now right click and press down arrow key to "Mange stored access polices" and press ENTER key.
10. TAB till "Add" button and press ENTER key.
11. TAB till "Delete" button and inspect it
12. Observe that Programmatic name is not given for "Delete" button.

## Actual Experience
Programmatic name is not given for "Delete" button

## Expected Experience
Programmatic name should be given for "Delete" button.

## User impact
Speech input and screen reader user will face difficulty because screen reader will announce as button only.

## Attachment
![Image](https://github.com/user-attachments/assets/e91ec45b-90e3-428a-9925-a8d9bf723833)

**Comment by InduPriya1805** [comment]
rev:inse

**Comment by craxal** [comment]
@kapilvaishna Fix is ready for validation.

**Comment by kapilvaishna** [comment]
@craxal issue is repro in build 1.37.0.(104). PFA for reference

![Image](https://github.com/user-attachments/assets/58eed44c-e1eb-46e0-9834-e1fa5b20496c)

**Comment by craxal** [comment]
@kapilvaishna Sorry, I forgot to send you a link to the new build. The build number you should have should be at least 20250128.2 or later. In the future, can you please report the build number that you test on?

**Comment by kupatkar99** [comment]
@craxal Verified the issue in below Environment:
Azure Storage Explorer version:1.38.0-dev(104)
Issue is not fixed. Please find below added attachment for reference.

![Image](https://github.com/user-attachments/assets/89060f06-c399-4c79-881c-46295f75012c)

**Comment by craxal** [comment]
@kupatkar99 The button is part of a table, and the column for the delete button should have an accessible name. Does the screen reader not announce that column when you focus on the button?

**Comment by kapilvaishna** [comment]
@craxal yes, name is required for control it will helpful both user screen reader and speech input user. 
Please find screen reader screen shot below 



![Image](https://github.com/user-attachments/assets/c19f2cf3-1b98-4371-a002-7a2086cfa05e)

**Comment by craxal** [comment]
@kupatkar99 Additional fixes merged. Please verify.

**Comment by kapilvaishna** [comment]
Issue is not repro in version 1.38.0-Dev (104) so closing the bug.

![Image](https://github.com/user-attachments/assets/d894fb6c-af38-4184-b569-c13ac786374a)
